product_configurator_mrp/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Product Configurator Manufacturing",
-    "version": "14.0.1.2.0",
+    "version": "14.0.1.2.1",
```

Comment:
No need to bump version, the merge bot will do that for you.

Reply:
Done the changes